backend/testUtils/tests.ts

```diff
@@ -136,10 +136,10 @@ export const assertResponseMatchesExpected = (
 ): void => {
   expect(result.id).not.toBeNull();
   expect(result.name).toEqual(expected.name);
-  expect(result.assessmentType).toEqual(expected.assessmentType.toString());
```

Comment:
is this change necessary?

Reply:
I added toString() in my previous PR because the enum used to be an integer. But now I changed the enum to be a string so I thought it's weird to toString() a string.

Reply:
ahh makes sense!